cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/internal/AliasedSSA.qll

```diff
@@ -234,6 +229,50 @@ Overlap getOverlap(MemoryLocation def, MemoryLocation use) {
   )
 }
 
+predicate isRelevantOffset(VirtualVariable vv, IntValue offset) {
```

Comment:
Does this predicate have to hold even for unknown offsets?

Reply:
It doesn't